github-auth-hook.js

```diff
@@ -22,7 +23,18 @@ passport.use(
                 // user can choose to not display any email, then use a default one as unleash required it
                 profile.emails.push(`${displayName}@unknown.com`);
             }
-            // Successful authentication, now check if the authenticated user is a member of the GH org/team
+            let user = new User({
+                name: profile.displayName,
+                email: profile.emails[0].value,
+            });
+
+            // Successful authentication, now check if the authenticated user is a member of the GH org/team, unless `dev mode` is enabled
```

Comment:
@xcoulon move this comment after dev-mode

Reply:
the comment ends with `...unless `dev mode` is enabled.` so I think it makes sense to leave it before the `if (devmode) {..}` check.